tests/test_config_propagation.py

```diff
@@ -41,12 +40,47 @@ def test_configuration():
         assert run_task.k8s_resources.requests == {"memory": "1024M", "cpu": "1"}
 
     assert run_task.labels == {"runner": "airflow"}
+    assert run_task.env_vars[0].to_dict() == {
+        "name": "EXAMPLE_ENV",
+        "value": "example",
+        "value_from": None,
+    }
+    assert run_task.env_vars[1].to_dict() == {
+        "name": "SECOND_EXAMPLE_ENV",
+        "value": "second",
+        "value_from": None,
+    }
     assert run_task.secrets == [
         Secret("env", "test", "snowflake-access-user-key", None),
         Secret("volume", "/var", "snowflake-access-user-key", None),
     ]
+    assert (
+        run_task.arguments[0]
+        == 'set -e; dbt --no-write-json run --target dev --vars "{}" --project-dir /dbt --profiles-dir /root/.dbt --select dim_users'  # noqa: E501
+    )
     assert run_task.config_file == "/usr/local/airflow/dags/kube_config.yaml"
     assert run_task.is_delete_operator_pod
     assert "--project-dir /dbt" in run_task.arguments[0]
     assert "--profiles-dir /root/.dbt" in run_task.arguments[0]
     assert "--target dev" in run_task.arguments[0]
+
+
+def test_configuration_with_qa_config():
```

Comment:
I feel like it should be a separate file for qa testing. It is not only configuration testing.